prototype_source/flight_recorder_tutorial.rst

```diff
@@ -169,9 +174,29 @@ To run the convenience script, follow these steps:
 
 2. To run the script, use this command:
 
-.. code:: python
+.. code:: shell
+
+  python fr_trace.py <dump dir containing trace files> [-o <output file>]
+
+If you install the PyTorch nightly build or build from scratch with ``USE_DISTRIBUTED=1``, you can directly use the following
+command directly:
+
+.. code:: shell
+
+  torchfrtrace <dump dir containing trace files> [-o <output file>]
+
+
+Currently, we support two modes for the analyzer script. The first mode allows the script to apply some heuristics to the parsed flight
+recorder dumps to generate a report identifying potential culprits for the timeout. The second mode is simply outputs the raw dumps.
+By default, the script prints flight recoder dumps for all ranks and all ``ProcessGroups``(PGs). This can be narrowed down to certain
+ranks and PGs. An example command is:
```

Comment:
Suggested clarification to sentence.

can be narrowed down to certain ranks and PGs using the `selected-ranks` option.